var/spack/repos/builtin/packages/parsec/package.py

```diff
@@ -59,6 +60,10 @@ class Parsec(CMakePackage, CudaPackage):
     # TODO: Spack does not handle cross-compilation atm
     # variant('xcompile', default=False, description='Cross compile')
 
+    depends_on("c", type="build")
+    depends_on("cxx", type="build")
+    depends_on("fortran", type="build")
```

Comment:
Can you explain why are we adding a dependency on `cxx` and `fortran`? How are they needed?

Reply:
Fortran is an optional dependency to generate the Fortran API module.